translations/en.i18n.json

```diff
@@ -27,6 +27,10 @@
     "unmute": {
       "unmute": "Unmute"
     },
+    "volume": {
+      "muted": "Muted. Click to unmute",
```

Comment:
is there a ticket for translations for additional languages ?

Reply:
no, what is the process in such case?

Reply:
@SivanA-Kaltura i created a ticket for it- https://kaltura.atlassian.net/browse/FEC-13023